docs/modules/rediscache.md

```diff
@@ -25,10 +24,24 @@ Provisions a new Redis cache.
 |----------------|------|-------------|----------|---------------|
 | `location` | `string` | The Azure region in which to provision applicable resources. | Required _unless_ an administrator has configured the broker itself with a default location. | The broker's default location, if configured. |
 | `resourceGroup` | `string` | The (new or existing) resource group with which to associate new resources. | N | If an administrator has configured the broker itself with a default resource group and nonde is specified, that default will be applied, otherwise, a new resource group will be created with a UUID as its name. |
+| `skuCapacity` | `integer` | The size of the Redis cache to deploy.  Valid values: for C (Basic/Standard) family (0, 1, 2, 3, 4, 5, 6), for P (Premium) family (1, 2, 3, 4). | N | If not provided, `0` is used for C (Basic/Standard) family; `1` is used for P (Premium) family. |
+| `enableNonSslPort ` | `string` | Specifies whether the non-ssl Redis server port (6379) is enabled. Valid values: (`enabled`, `disabled`) | N | If not provided, `enabled` is used. **Note**:  this behavior is different from Azure portal. In OSBA, non-SSL port is enabled by default. That's because we want to make sure your application can work normally even your application doesn't support Redis with SSL. |
```

Comment:
I lean towards not enabling the unsecured port by default. This kind of goes against the Microsoft value of "secure by default". If someone wants to use the unsecured port, then they have this flag here to do so, no?

Reply:
That's configured to be enabled to keep consistent with [current implementation](https://github.com/Azure/open-service-broker-azure/blob/master/pkg/services/rediscache/arm_template.go#L20). I agree it's reasonable to make insecure connection disabled by default. I will change the code later.